src/mad_spark_alt/core/adaptive_prompts.py

```
@@ -0,0 +1,541 @@
+            "questioning": {
+                "regular": """As a technical architect, generate 2 strategic technical questions about: "{prompt}"
+{previous_insights}
+Focus on scalability, maintainability, and technical feasibility.
+Consider architecture decisions, technology choices, and implementation challenges.
+Format each question on a new line starting with "Q:".""",
+                
+                "concrete": """As a senior software engineer, generate 2 practical implementation questions about: "{prompt}"
+{previous_insights}
+Focus on specific technical requirements, development tools, and deployment concerns.
+Consider performance, security, and integration requirements.
+Format each question on a new line starting with "Q:"."""
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)